data/mod3.py

```diff
@@ -0,0 +1,91 @@
+fe3 = [
+      {
+        "htmlLink": "https://www.google.com/calendar/event?eid=Nmd1OG1qamM5czliYWdiMTF2aHBjaTE0MWZfMjAyMDA5MThUMjAxNTAwWiBjYXNpbWlyY3JlYXRpdmUuY29tX2tyYjlwMzVjazM1bTR1b2ppNWQyNzE1ODQ0QGc",
+        "summary": "Code Fair",
+        "location": "Zoom links shared in #announcements",
+        "start": "2020-09-18T20:15:00Z",
```

Comment:
`fe3` has these timestamps that end in `Z`, which means the time listed is at Universal time and needs to be offset.